### ISSUE TITLE
Upgrade to RxJS 6 or add rxjs-compat dependency

### ISSUE THREAD

**Issue by johngrogg** [kondi/rxjs-grpc#29]
When using rxjs-grpc with RxJS 6, rxjs-compat is required because of the use of the following import pattern:
```
import { Observable } from "rxjs/Observable";
```

With RxJS 6 it should be updated to:
```
import { Observable } from "rxjs";
```

See https://github.com/ReactiveX/rxjs/issues/3764 for more details

**Comment by FedeBev** [comment]
Watch if my solve your issue PR #35!

**Comment by kondi** [comment]
If you need rxjs6 support, please use rxjs-grpc@rxjs6 prerelease version.
From version 1.0-rc.0 it will be officially supported, however, there will be other breaking changes as well.

**Comment by redevill** [comment]
Internally to the project, with a fresh npm install (V6.3.3) the node-modules/rxjs/util folder still refers to the rxjs-compat.

**Comment by kondi** [comment]
Released in [v0.2.0](https://github.com/kondi/rxjs-grpc/releases/tag/v0.2.0).
